### DISCUSSION TITLE
How to add multiple accounts to one station location?

### DISCUSSION THREAD

**Discussion by Piter-NS** [magicbug/Cloudlog#1845] [Q&A]
Hello everyone,
First of all, I would like to thank the authors for the great job they are doing in this project. A very nice online logbook.

Currently in the club we use version 1.9.9 of Cloudlog and this log supports one location to which QSOs are added by several operators. Each operator has separate access to this logbook using their private call sign as login. All QSOs under the club call sign are stored in one location. In the latest version, i.e. 2.2 installed for testing, the location of the station must be assigned to one operator (login). Is it possible to share this location with other operators and use the new version in the same way as version 1.9.9?

**Comment by szporwolik** [comment]
I'm new to Cloudlog, great piece of software. I'm also checking it for possible usage in the Club.
How to set it up, so multiple operators can work one callsign/logbook ?
